packages/shared-core/package.json

```diff
@@ -2,21 +2,28 @@
   "name": "@budibase/shared-core",
   "version": "2.4.8-alpha.1",
   "description": "Shared data utils",
-  "main": "dist/src/index.js",
-  "module": "dist/src/index.js",
-  "types": "dist/src/index.d.ts",
+  "main": "dist/mjs/src/index.js",
```

Comment:
Normally I would say that the `main` key should probably point to the CJS build, but when this module is intended to be consumed via Typescript from node environments maybe that changes things.